common/pom.xml

```diff
@@ -6,14 +6,21 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../pom.client.xml</relativePath>
+  </parent>
+
   <groupId>com.azure</groupId>
   <artifactId>azure-common-parent</artifactId>
   <version>1.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
 
-  <name>Azure Common Libraries for Java</name>
+  <name>Microsoft Azure Common Libraries for Java</name>
   <description>This package contains the common libraries Java clients.</description>
```

Comment:
This package contains the common libraries `for` Java clients.

Reply:
```suggestion
  <description>This package contains the common libraries for Java clients.</description>
```